tools/accuracy_checker/configs/ssd_resnet50_v1_fpn_coco.yml

```diff
@@ -0,0 +1,44 @@
+models:
+  - name: ssd_resnet50_v1_fpn_coco
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   public/ssd_resnet50_v1_fpn_coco/FP32/ssd_resnet50_v1_fpn_coco.xml
+        weights: public/ssd_resnet50_v1_fpn_coco/FP32/ssd_resnet50_v1_fpn_coco.bin
+        adapter: ssd
+
+      - framework: dlsdk
+        tags:
+          - FP16
+        model:   public/ssd_resnet50_v1_fpn_coco/FP16/ssd_resnet50_v1_fpn_coco.xml
+        weights: public/ssd_resnet50_v1_fpn_coco/FP16/ssd_resnet50_v1_fpn_coco.bin
+        adapter: ssd
+
+      - framework: dlsdk
+        tags:
+          - FP32-INT8
+        model:   public/ssd_resnet50_v1_fpn_coco/FP32-INT8/ssd_resnet50_v1_fpn_coco.xml
+        weights: public/ssd_resnet50_v1_fpn_coco/FP32-INT8/ssd_resnet50_v1_fpn_coco.bin
+        adapter: ssd
+
+      - framework: dlsdk
+        tags:
+          - FP16-INT8
+        model:   public/ssd_resnet50_v1_fpn_coco/FP16-INT8/ssd_resnet50_v1_fpn_coco.xml
+        weights: public/ssd_resnet50_v1_fpn_coco/FP16-INT8/ssd_resnet50_v1_fpn_coco.bin
```

Comment:
should be removed, we do not provide int8 models for public models

Reply:
Done